CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ project(${PROJECT_NAME} VERSION ${PROJECT_VERSION} LANGUAGES C CXX)
 
 include(${CMAKE_CONFIG_DIR}/common.cmake)
 
-option(BUILD_TESTS "Build examples" ON)
+option(BUILD_TESTS "Build examples")
```

Comment:
Let's reserve `BUILD_TESTS` for unit tests and `BUILD_EXAMPLES` for examples.

Reply:
I added `BUILD_EXAMPLES` and kept `BUILD_TESTS` for later use.